ssvsigner/client/client.go

```diff
@@ -0,0 +1,252 @@
+package ssvsignerclient
+
+import (
+	"bytes"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+	"time"
+
+	"go.uber.org/zap"
+
+	"github.com/ssvlabs/ssv/ssvsigner/server"
+	"github.com/ssvlabs/ssv/ssvsigner/web3signer"
+)
+
+type Status = server.Status
+
+const (
+	StatusImported   Status = "imported"
+	StatusDuplicated Status = "duplicate"
+	StatusDeleted    Status = "deleted"
+)
+
+type ShareDecryptionError error
```

Comment:
Right now, `ShareDecryptionError` is never used in the rest of the code. Do we need it at all? If not, we should consider removing it.